runtime/src/version.rs

```diff
@@ -8,13 +8,13 @@ use sp_version::RuntimeVersion;
 pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("creditcoin-node"),
 	impl_name: create_runtime_str!("creditcoin-node"),
-	authoring_version: 2,
+	authoring_version: 3,
```

Comment:
For record-keeping:

> Just the spec version, the authoring version only needs to be bumped when you make a change to the runtime that affects authoring blocks. Since your PR just adds pallets that add functionality it isn’t needed